packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -12,9 +12,9 @@ const ChatAdapterExampleText = require('!!raw-loader!./snippets/ChatAdapterExamp
 
 Composites deliver turn-key communication experiences for developers.
 They make it easy for developers to add end-to-end communication experiences to their apps with only a couple lines of code.
-To power Composites, the UI library provides `Adapters`, one for the calling composite [here](./?path=/docs/composites-call-basicexample--basic-example) and one for chat composite
+To power Composites, the UI library provides `Adapters`, one for the `CallComposite` [here](./?path=/docs/composites-call-basicexample--basic-example) and one for `ChatComposite`
 [here](./?path=/docs/composites-chat-basicexample--basic-example).
-These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like usePropsFor to provide developers a single interface to configure Composites.
```

Comment:
I'm torn whether or not Components and Composites are proper nouns (and hence should be capitalized) as they refer to _our_ UI Library Components (specific entity) and not components as a general term
e.g. here when we say `configure composites`, we are shortening what we are really saying: `configure UI Library Composites` but my grammar isn't good enough to know if that constitutes a capital letter or not
Do you know which it should be or does the Microsoft best practices mention this at all?